gapis/api/vulkan/vulkan.go

```diff
@@ -291,7 +307,41 @@ func (API) ResolveSynchronization(ctx context.Context, d *sync.Data, c *path.Cap
 			case VkCmdDebugMarkerEndEXTArgsʳ:
 				popMarker(DebugMarker, uint64(i), nCommands)
 			}
+
+			// Handle draw commands grouping.
+			cmdName := ""
+			if generatingId != nil {
+				cmdName = generatingId.CmdName()
+			}
+			isDrawCmd := strings.HasPrefix(cmdName, "vkCmdDraw") || strings.HasPrefix(cmdName, "vkCmdDispatch")
```

Comment:
We should use flags to get these.

Reply:
Alternatively, we could fold this logic into the switch above and then we don't care so much about names at all -- we'll look at the command reference types instead

Reply:
After discussing with Chris offline, we decided the judging logic for isDrawCmd value can be kept the same. 
1. We abandoned the use of api.cmd for safety against MEC case, so no flags here.
2. The switch option will result in enlisting a whole bunch of types, like all the "vkCmdSet*" commands.